docs/source/installation.rst

```diff
@@ -153,37 +158,32 @@ The ``.whl`` package and ``.exe`` installers can be built with:
 
 The resulting packages are generated in the ``dist/`` folder.
 
+Contributor or 3rd party extension developer setup
+--------------------------------------------------
 
-sktime build requirements
--------------------------
-
-This section outlines the ``sktime`` build requirements. These are required for:
-
-* installing ``sktime`` from source, e.g., development versions
-* the advanced developer set-up
-
-
-Setting up a development environment
-------------------------------------
+1. Follow the Git workflow: Fork and clone the repository as described in [Git and GitHub workflow](https://www.sktime.net/en/stable/developer_guide/git_workflow.html)
 
-First set up a new virtual environment. Our instructions will go through the commands to set up a ``conda`` environment which is recommended for sktime development.
+2. Set up a new virtual environment. Our instructions will go through the commands to set up a ``conda`` environment which is recommended for sktime development.
 This relies on an `anaconda installation <https://www.anaconda.com/products/individual#windows>`_. The process will be similar for ``venv`` or other virtual environment managers.
 
 In the ``anaconda prompt`` terminal:
 
-1. Navigate to your local sktime folder :code:`cd sktime`
+3. Navigate to your local sktime folder, :code:`cd sktime` or similar
 
-2. Create new environment with python 3.8: :code:`conda create -n sktime-dev python=3.8`
+4. Create new environment with a supported python version: :code:`conda create -n sktime-dev python=3.8` (or :code:`python=3.11` etc)
 
    .. warning::
        If you already have an environment called "sktime-dev" from a previous attempt you will first need to remove this.
 
-3. Activate the environment: :code:`conda activate sktime-dev`
+5. Activate the environment: :code:`conda activate sktime-dev`
 
-4. Build an editable version of sktime :code:`pip install -e .[dev]`
-If you also want to install the soft dependencies, install them one-by-one or use: :code:`pip install -e .[all_extras,dev]`
+6. Build an editable version of sktime :code:`pip install -e .[dev]`
+In order to install all soft dependencies, use :code:`pip install -e .[dev]`
```

Comment:
Is it a correct statement? Do we not require `all_extras`, as said in just below line?

Reply:
well, it was inconsnistent between the cases, and I thought we don't strictly need `all_extras` for a developer setup (I do't have it most of the time)